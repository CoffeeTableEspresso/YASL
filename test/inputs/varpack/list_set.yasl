
fn cycle(x, y, z) {
    return y, z, x
}

const ls = [ .first, .second, .third ]

const a, ls[1], const c = 'extra', cycle('x', 'y', 'z')

echo a
echo c
echo ls
