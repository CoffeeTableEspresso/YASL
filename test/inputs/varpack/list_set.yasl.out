extra
z
[first, y, third]
